CHANGELOG.md

```diff
@@ -258,22 +259,22 @@ Rolled in recent fixes, testing improvements, and dependency updates for the Cat
 - Bump thiserror from 1.0.19 to 1.0.20 [\#2376](https://github.com/input-output-hk/jormungandr/pull/2376)
 - Bump base64 from 0.12.1 to 0.12.2 [\#2375](https://github.com/input-output-hk/jormungandr/pull/2375)
 - Disable double logging in staging tests [\#2372](https://github.com/input-output-hk/jormungandr/pull/2372)
-- \[Test\] fix test\_legacy\_node\_all\_fragments test. [\#2371](https://github.com/input-output-hk/jormungandr/pull/2371)
+- \[Test\] fix test_legacy_node_all_fragments test. [\#2371](https://github.com/input-output-hk/jormungandr/pull/2371)
```

Comment:
why this?

Reply:
oops I have used VS code formatter on this file, reverting it

Reply:
fixed